core/src/main/java/io/questdb/cairo/TxReader.java

```diff
@@ -312,23 +339,23 @@ public boolean unsafeLoadAll() {
     }
 
     public boolean unsafeLoadBaseOffset() {
-        this.version = unsafeReadVersion();
+        version = unsafeReadVersion();
         Unsafe.getUnsafe().loadFence();
 
         boolean isA = (version & 1L) == 0L;
-        this.baseOffset = isA ? roTxMemBase.getInt(TX_BASE_OFFSET_A_32) : roTxMemBase.getInt(TX_BASE_OFFSET_B_32);
-        this.symbolsSize = isA ? roTxMemBase.getInt(TX_BASE_OFFSET_SYMBOLS_SIZE_A_32) : roTxMemBase.getInt(TX_BASE_OFFSET_SYMBOLS_SIZE_B_32);
-        this.partitionSegmentSize = isA ? roTxMemBase.getInt(TX_BASE_OFFSET_PARTITIONS_SIZE_A_32) : roTxMemBase.getInt(TX_BASE_OFFSET_PARTITIONS_SIZE_B_32);
+        baseOffset = isA ? roTxMemBase.getInt(TX_BASE_OFFSET_A_32) : roTxMemBase.getInt(TX_BASE_OFFSET_B_32);
+        symbolsSize = isA ? roTxMemBase.getInt(TX_BASE_OFFSET_SYMBOLS_SIZE_A_32) : roTxMemBase.getInt(TX_BASE_OFFSET_SYMBOLS_SIZE_B_32);
+        partitionSegmentSize = isA ? roTxMemBase.getInt(TX_BASE_OFFSET_PARTITIONS_SIZE_A_32) : roTxMemBase.getInt(TX_BASE_OFFSET_PARTITIONS_SIZE_B_32);
 
         // Before extending file, check that values read are not dirty
         Unsafe.getUnsafe().loadFence();
         if (unsafeReadVersion() != version) {
             return false;
         }
 
-        this.size = calculateTxRecordSize(symbolsSize, partitionSegmentSize);
```

Comment:
Is it a code inspection rule? Can you share it with the team so that we all on the same coding style?

Reply:
TxReader line 383:

```
private void clearData() {
        baseOffset = 0;
        size = 0L;
        partitionTableVersion = -1L;
        attachedPartitionsSize = -1;
        attachedPartitions.clear();
        version = -1L;
        txn = -1L;
        seqTxn = -1L;
    }
```

the presence of the `L` behind the `=` tells me that the variable is of type `long`, instead of `int`, `byte`, `banana`, without having to scroll all the way to the definition, or worse, have to hover with the mouse over the vatiable... this is all because I am lazy, but not world class level lazy as those who do not append the `L`s where they help :P